boot_code/Makefile

```diff
@@ -18,7 +18,7 @@
 CC      = riscv32-unknown-elf-gcc
 OBJCOPY = objcopy
 
-CFLAGS   = -Os -g -fno-jump-tables -march=rv32imfcxpulpv2
+CFLAGS   = -Os -g -fno-jump-tables -march=rv32imc -UARCHI_CORE_HAS_PULPV2 -DRV_ISA_RV32
```

Comment:
What is `RV_ISA_RV32` for again? Looks like anothe macro name from pulp-sdk/pulp-runtime that is non-descriptive.

Reply:
`RV_32_ISA` is one of the multiple ways to signify not to use custom pulp instructions, i.e. only use rv32imc. I copied these settings from the pulp-runtime modifications, there are a few code sections wrapped with this macro and ifdef there.